src/components/structures/TimelinePanel.js

```diff
@@ -414,9 +417,10 @@ var TimelinePanel = React.createClass({
                 } else if(lastEv && this.getReadMarkerPosition() === 0) {
                     // we know we're stuckAtBottom, so we can advance the RM
                     // immediately, to save a later render cycle
+
+                    // This call will setState with readMarkerEventId = lastEv.getId()
```

Comment:
... except that the `true` argument is called `inhibitSetState` - so I don't think it will.